<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Help</class>
 <widget class="QDialog" name="Help">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1120</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1120</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 50);</string>
  </property>
  <widget class="QLabel" name="__help">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>400</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;帮 助&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="__help_1">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>240</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>基 本 玩 法</string>
   </property>
  </widget>
  <widget class="QPushButton" name="__help_2">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>320</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>限 时 模 式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="__help_3">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>400</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>双 人 模 式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="__help_4">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>480</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>商 店</string>
   </property>
  </widget>
  <widget class="QPushButton" name="__help_5">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>560</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>返 回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="__right">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>250</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 75 20pt &quot;Agency FB&quot;;
background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="__left">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>300</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 75 20pt &quot;Agency FB&quot;;
background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="__back">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>350</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 75 20pt &quot;Agency FB&quot;;
background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>《 </string>
   </property>
  </widget>
  <widget class="QLabel" name="__h_1_1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>700</width>
     <height>446</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-1-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_1_2">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>160</y>
     <width>770</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-1-2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_1_3">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>160</y>
     <width>750</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-1-3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_1_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>740</width>
     <height>453</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-1-4.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_1_5">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>170</y>
     <width>730</width>
     <height>445</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-1-5.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_1_6">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>160</y>
     <width>750</width>
     <height>457</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-1-6.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_1_7">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>170</y>
     <width>750</width>
     <height>446</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-1-7.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_1_8">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>170</y>
     <width>710</width>
     <height>449</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-1-8.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_2_1">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>160</y>
     <width>730</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-2-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_3_1">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>160</y>
     <width>730</width>
     <height>452</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-3-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_4_1">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>160</y>
     <width>650</width>
     <height>458</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-4-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_4_2">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>165</y>
     <width>730</width>
     <height>447</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-4-2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_4_3">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>160</y>
     <width>730</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-4-3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_4_4">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>160</y>
     <width>730</width>
     <height>452</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-4-4.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_4_5">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>160</y>
     <width>730</width>
     <height>452</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-4-5.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_4_6">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>160</y>
     <width>730</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-4-6.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_4_7">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>160</y>
     <width>730</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-4-7.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_4_8">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>160</y>
     <width>730</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-4-8.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_4_9">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>160</y>
     <width>730</width>
     <height>449</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-4-9.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_4_10">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>160</y>
     <width>710</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-4-10.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_4_11">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>720</width>
     <height>465</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-4-11.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__h_4_12">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>740</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/h-4-12.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
